### ISSUE TITLE
How do enable dark mode? 

### ISSUE THREAD

**Issue by WingCH** [xmartlabs/Eureka#1919]
I have search dark mode in issues page, but dark mode seems is auto-enabled, but the color is incorrect in dark mode.

```swift
class SettingsViewController: FormViewController {
    override func viewDidLoad() {
        super.viewDidLoad()
        form +++ Section("Header")
           <<< ActionSheetRow<String>() {
                      $0.title = "title"
                       $0.selectorTitle = "selectorTitle"
                       $0.options = ["a","b"]
                       $0.value = "a"
           }
    }
}
```

![2019-09-30_23-45-18](https://user-images.githubusercontent.com/19588517/65894534-87132580-e3dc-11e9-8fe8-ade0dbe8b027.png)

**Comment by mats-claassen** [comment]
This is fixed in master but not released yet. Are you using master version?

**Comment by WingCH** [comment]
Thanks your reply!
i use cocopads install the libray

```
pod 'Eureka'
```

the version is `5.0.1`

**Comment by rorycoconnor** [comment]
@mats-claassen When will this be released? I am also using 5.0.1.

**Comment by mats-claassen** [comment]
Released in [5.1.0](https://github.com/xmartlabs/Eureka/releases)
